src/test/java/uk/gov/digital/ho/hocs/info/api/UserResourceRestTest.java

```diff
@@ -32,7 +32,7 @@
 
   @Test
   public void testCreateUserWithWhitelistedDomains() throws Exception {
-    List<String> whitelistedDomains = List.of("mick@homeoffice.gov.uk", "mick@hmpo.gov.uk");
+    List<String> whitelistedDomains = List.of("mick@example.co.uk", "mick@example.org");
```

Comment:
Incidentally might be nice to have some tests that check if subdomains are let through

Reply:
There is a new test file added that tests the validator more explicitely. This contains a test for the subdomains to ensure they are not part of this.